thinc/backends/ops.py

```diff
@@ -602,7 +602,16 @@ def dtanh(self, Y: FloatsT, *, inplace: bool = False) -> FloatsT:
         else:
             return 1 - Y ** 2
 
-    def softmax(self, x: FloatsT, *, inplace: bool = False, axis: int = -1) -> FloatsT:
+    def softmax(
+        self,
+        x: FloatsT,
+        *,
+        inplace: bool = False,
+        axis: int = -1,
+        temperature: float = 1.0,
+    ) -> FloatsT:
+        if temperature != 1.0:
+            x = x / temperature
```

Comment:
We need to guard against `temperature` 0.0 as well - probably by throwing an error?

Reply:
Yeah, that would be good to validate. We should probably also reject negative values.

Fixed and added test.